src/websockets/http.py

```diff
@@ -292,6 +292,16 @@ def copy(self) -> "Headers":
         copy._list = self._list.copy()
         return copy
 
+    def get_safe(self, header: str) -> str:
+        from .exceptions import InvalidHeader
```

Comment:
I guess you've had to do this to avoid an import cycle. This is a very common issue since I implemented static type checking with mypy.

The `websockets.exceptions` module is imported at the bottom at `websockets.http`, so you remove this import use `websockets.exceptions.InvalidHeader`.

It's ugly but I'd rather have a module-level import at the bottom of a module than scattered imports inside functions.

But if you're moving the code to websockets.handshake, the problem goes away.